common/libs/nodes/infura/index.js

```diff
@@ -0,0 +1,11 @@
+import RPCNode from '../rpc';
+import InfuraClient from './client';
+
+export default class InfuraNode extends RPCNode {
+  client: InfuraClient;
+
+  constructor(endpoint: string) {
+    super(endpoint);
+    this.client = new InfuraClient(endpoint);
```

Comment:
Shouldn't it work fine if you remove the constructor completely? Default constructor should take care of this I believe

Reply:
Default constructor loads the RPC client, I need the InfuraClient here.

Reply:
Ah you're right. My bad